vos/vos.py

```diff
@@ -1452,22 +1461,33 @@ def copy(self, source, destination, send_md5=False):
                     response = self.conn.session.get(get_url, timeout=(2, 5), stream=True)
                     source_md5 = response.headers.get('Content-MD5', source_md5)
                     response.raise_for_status()
-                    with open(destination, 'w') as fout:
+                    # Write to a temporary file instead of destination.
+                    if destination[0:4] == 'vos:':
+                        tobj = NamedTemporaryFile(prefix='vospace.')
```

Comment:
Where is the temporary file removed? Docs says that with the delete arg set to True the file is deleted as soon as it's closed but otherwise, I think you need to do it.